spec/board_helper.rb

```diff
@@ -0,0 +1,45 @@
+class BoardHelper
+  def initialize(board)
+    @board = board
+  end
+
+  def string_to_board(string_board)
+    rows = string_board.split(",")
+
+    @board.board.size.times do |index_row|
```

Comment:
This looks rather complicated. Isn't it possible to just use the public API of your board? As in having subsequent calls to `set_mark` while you're looping over the string?